operations/deployment/ansible/tasks/mount.yml

```diff
@@ -0,0 +1,63 @@
+# Mount EFS
+
+- name: Install required system packages
+  apt: 
+    name: "{{ item }}" 
+    state: present 
+    update_cache: yes
+    cache_valid_time: 3600 # 1 hour
+  loop:
+    - 'nfs-kernel-server'
+    - 'git'
+    - 'binutils'
+
+- name: Check if amazon-efs-utils is present
+  stat:
+    path: /home/ubuntu/efs-utils
+  register: check_efs_utils
+
+- name: Cloning amazon-efs-utils from GitHub
+  ansible.builtin.git:
+    repo: 'https://github.com/aws/efs-utils'
+    dest: /home/ubuntu/efs-utils
+    update: false
+  become: yes
+
+- name: Build amazon-efs-utils
+  ansible.builtin.shell: 
+    cmd: |
+      ./build-deb.sh
+      apt-get -y install ./build/amazon-efs-utils*deb
+    chdir: /home/ubuntu/efs-utils
+  become: yes
+  when: not check_efs_utils.stat.exists
```

Comment:
imo, everything above here (all the install/build stuff) should go in a different task file (maybe `install` ?)
that way, the only thing in mount is the stuff that's specifically mounting

Reply:
So my reasoning of putting this in here is; 
- If mounting doesn't take place then no time is used prepping the environment for mounting

Reply:
I disagree personally - If the user isn't going to use the mount task, then I don't see a point in adding to their deployment time by installing / building unnecessary packages.

Reply:
I agree with that. 

i think the name of the file is throwing me off. it's not "mounting". it's "installing and mounting".  I agree that `install` ins't the right place, but perhaps a `install_efs_deps` file would be?

either that or maybe rename the file to `install_and_mount`

also, this is probably a nit at this point, so don't let it hold up the PR